dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/HostsUpdater.java

```diff
@@ -15,54 +15,102 @@
  */
 package com.netflix.dyno.connectionpool.impl;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
+import java.util.*;
```

Comment:
Do we really need all the APIs from the util package?

Reply:
editor does it, It is not really going to import all of them unless we use them